pepper-apis/src/main/resources/db-changes/schema/DDP-4854-increase-i18nstudyactivity-title-len.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
increasing title column length to accommodate both title and subtitle. Looks like testboston front-end cannot handle subtitle separately and need to be in title.